modules/codeready-workspaces.adoc

```diff
@@ -4,15 +4,15 @@
 // * adding_service_cluster/rosa-available-services.adoc
 
 [id="codeready-workspaces_{context}"]
-= Red Hat CodeReady Workspaces
+= {openshift-dev-spaces-productname}
```

Comment:
It looks like this module is not included in any assemblies in `main`. Given this, I can't review a preview of how the updated attributes would render if the module was made live. But, given the use of the same attributes in `welcome/oke_about.adoc`, I think they look ok.